Sources/CacheAdvance/CacheReader.swift

```diff
@@ -75,16 +72,8 @@ final class CacheReader {
             return message
 
         case .emptyRead:
-            guard offsetInFileAtEndOfNewestMessage < startingOffset else {
-                // We started reading before the offset of the end of the newest message, therefore we expected a message to be read. We instead read an empty space, meaning that the file is corrupt.
-                throw CacheAdvanceError.fileCorrupted
-            }
-
-            // We know the next message is at the end of the file header. Let's seek to it.
-            try reader.seek(to: FileHeader.expectedEndOfHeaderInFile)
```

Comment:
I agree that the library is safer against unknown unknown errors without this line of code.